src/main/java/InputReview.java

```diff
@@ -0,0 +1,46 @@
+import java.util.Scanner;
+
+public class InputReview {
+
+    private int cars;
+    private int attempts;
+    private static final Scanner scanner = new Scanner(System.in);
+
+    public InputReview() {
+    }
+
+    public InputReview(int cars, int attempts) {
+        this.cars = cars;
+        this.attempts = attempts;
+    }
+
+    public void inputCarNumber() {
+
+        System.out.println("자동차 대수는 몇 대 인가요?");
+        try {
+            this.cars = scanner.nextInt();
+        } catch (Exception e) {
+            throw new NotNumberException("숫자만 입력하세요!");
+        }
+        if (this.cars < 0) throw new NotPositiveNumberException("양수만 입력하세요");
```

Comment:
```suggestion
        if (this.cars < 0) {
            throw new NotPositiveNumberException("양수만 입력하세요");
        }
```
중괄호를 생략할 때, 자잘한 실수가 발생할 수 있으니, 항상 중괄호를 사용하시는걸 추천드립니다ㅎㅎ